src/main/java/frc/robot/subsystems/swerve/module/hardware/HardwareModule.java

```diff
@@ -0,0 +1,163 @@
+package frc.robot.subsystems.swerve.module.hardware;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import frc.robot.hardware.angleencoder.IAngleEncoder;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.subsystems.swerve.module.Module;
+import frc.robot.subsystems.swerve.module.ModuleConstants;
+import frc.robot.subsystems.swerve.module.ModuleUtils;
+import frc.robot.subsystems.swerve.module.stuffs.DriveStuff;
+import frc.robot.subsystems.swerve.module.stuffs.EncoderStuff;
+import frc.robot.subsystems.swerve.module.stuffs.SteerStuff;
+import frc.utils.Conversions;
+import frc.utils.calibration.sysid.SysIdCalibrator;
+import org.littletonrobotics.junction.AutoLog;
+import org.littletonrobotics.junction.Logger;
+
+public class HardwareModule extends Module {
+
+	@AutoLog
+	public static class CouplingInputs {
+
+		public Rotation2d uncoupledVelocityPerSecond = new Rotation2d();
+		public Rotation2d[] uncoupledPositions = new Rotation2d[0];
+
+	}
```

Comment:
Same here, it's fine if we decide it would be a part of the module, but we need to be consistent -- and right now we're doing the opposite.